cypress/integration/homePage.js

```diff
@@ -0,0 +1,210 @@
+import faker from "faker";
+
+import Customer from "../apiRequests/Customer";
+import { HOMEPAGE_SELECTORS } from "../elements/homePage/homePage-selectors";
+import HomePageSteps from "../steps/homePageSteps";
+import { urlList } from "../url/urlList";
+import ChannelsUtils from "../utils/channelsUtils";
+import HomePageUtils from "../utils/homePageUtils";
+import OrdersUtils from "../utils/ordersUtils";
+import ProductsUtils from "../utils/productsUtils";
+import ShippingUtils from "../utils/shippingUtils";
+
+// <reference types="cypress" />
+describe("User authorization", () => {
+  const startsWith = "Cy-";
+
+  const customer = new Customer();
+  const productsUtils = new ProductsUtils();
+  const shippingUtils = new ShippingUtils();
+  const ordersUtils = new OrdersUtils();
+  const channelsUtils = new ChannelsUtils();
+  const homePageUtils = new HomePageUtils();
+  const homePageSteps = new HomePageSteps();
+
+  let customerId;
```

Comment:
`let`s complicate understanding the code. Why not use awaits and retrieve values straight from there?

Reply:
Cypress doesn't recommended to use async/await commands. These commands don't behave as expected when running code on cypress.